### PR TITLE
Kiran mehta day 6

### PR DESCRIPTION
**Pull request by Kiran-Mehta**

### CODE REVIEW

File: Day 6/DisplayPost.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import Post from './Post'
+import Button from './components/Button';
+
+const DisplayPost = () => {
+
```
Comment (bhandwalkar) [inline Day 6/DisplayPost.js:7]: The component name rarely / never starts with a verb. The name of the component is usually a Noun based on what it does. Ideally, the name should be.

Posts
PostList

File: Day 6/Post.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                <p>
+                    <strong>Title:</strong> {title}
+                </p>
+                <p>
+                    <strong ></strong> {body}
+                </p>
+                <Button onClick={handleClick} label={"Comments"} length={comments.length}/>
+                {loading && <h1> Post and Comments are Loading...</h1>}
+                {showComments && <Comments showComments={showComments} comments={comments}/>}
+                <hr />
```
Comment (bhandwalkar) [inline Day 6/Post.js:43]: Very unreadable code format. 

```
  <Button onClick={handleClick} label={"Comments"} length={comments.length}/>
   {
         loading && (<h1> Post and Comments are Loading...</h1>)
   }
   {
       showComments && (<Comments showComments={showComments} comments={comments}/>)
   }
```

**Review by bhandwalkar — CHANGES_REQUESTED** [review]

**Comment by bhandwalkar** [comment]
@Kiran-Mehta How are you uploading the files, there is no package.json, I can't test your app.

File: Day 6/DisplayPost.js (hunk 1 above)
Comment (Kiran-Mehta) [reply Day 6/DisplayPost.js:7]: Resolved.

**Pull request merged**